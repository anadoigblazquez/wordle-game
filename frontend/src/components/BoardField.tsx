
const BoardField = () => {
  return (
    <div className="
      h-[62px] 
      w-[62px] 
      border
      border-gray-200
      flex
      justify-center
      content-center
      flex-wrap
      text-2xl
      font-bold"></div>
  )
}

export default BoardField