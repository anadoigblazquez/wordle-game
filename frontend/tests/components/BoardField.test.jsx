import { render } from '@testing-library/react';
import BoardField from '../../src/components/BoardField';

describe('Tests on BoardField', () => {
    const className = 'className';

    test('Should match snapshot', () => {
        const { container } = render( <BoardField className={className} /> );
        expect( container ).toMatchSnapshot();
    })
    
})