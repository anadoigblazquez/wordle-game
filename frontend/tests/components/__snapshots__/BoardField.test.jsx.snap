// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on BoardField Should match snapshot 1`] = `
<div>
  <div
    class=" h-[62px]  w-[62px]  border border-gray-200 flex justify-center content-center flex-wrap text-2xl font-bold"
  >
    L
  </div>
</div>
`;
